openedx/core/djangoapps/credit/urls.py

```diff
@@ -4,11 +4,17 @@
 from django.conf.urls import patterns, url
 
 from .views import create_credit_request, credit_provider_callback
+from .api.provider import get_credit_provider_info
 
 urlpatterns = patterns(
     '',
 
     url(
+        r"^v1/provider/(?P<provider_id>[^/]+)/$",
```

Comment:
We will have slight change in url patterns as part of PR's: https://github.com/edx/ecommerce/pull/192/, https://github.com/edx/edx-platform/pull/8853/